File: code_review/work/kato/PHASE3/step1/TravelPrice.php
<?php
class TravelPrice
{
    private $total_price = 0;
    private $prices = [];

    public function __construct($round_trip_price, $hotel_price)
    {
        $this->prices[] = $round_trip_price;
        $this->prices[] = $hotel_price;
    }

    public function getTotalPrice()
    {
        $this->total_price = array_sum($this->prices);
        return $this->total_price;
    }
}

File: code_review/question/PHASE2/omija/step2/travel-price.rb
# 旅費合計を算出するクラス
class TravelPrice
    def initialize(round_trip_price, hotel_price)
      @round_trip_price = round_trip_price
      @hotel_price = hotel_price
    end
  
    # 旅費合計を算出するメソッド
    def total_price
      puts "交通費: #{@round_trip_price}円"
      puts "ホテル代: #{@hotel_price}円"
      puts "旅費合計: #{@round_trip_price + @hotel_price}円"
    end
  end